src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,23 @@
 
 public class Application {
     public static void main(String[] args) {
+        Ball[] balls = generateBalls(3);
```

Comment:
Let's simplify your logic.
You should create three balls using getRandomBall() method from Lottery class. And print information about them in the console.
You don't need to create an array of balls because it takes a place in memory that is redundant. Also, to optimise performance you don't need to create two loops. 

Try to avoid magic numbers in your code.